### ISSUE TITLE
Notes from data entry process

### ISSUE THREAD

**Issue by colinxfleming** [colinxfleming/dcaf_case_management#615]
A core team of @verbingthenoun  and @NerdyGirl537 (plus a short assist from myself) did a slew of seed data entry. Wanted to make sure we logged these interactions with the system.

What follows are a couple notes I jotted down during the process:
* I entered a patient who had a phone extension. Since we strictly format phone to 10char, we couldn't squeeze this in at the end. I wound up putting it down as other contact, but this is a place where the current phone stuff kinda stretches. (#641)
* The sheet has clinic names which sometimes encompass multiple clinics (eg PP of metro washington includes DC and Silver Spring). Wasn't clear to me whether there were any cases where the distinction was important. (#642)
* Probably it's important to pop up the new patient form even if there are results, right? Had a couple patients with similar first names and had to stop myself from making an error a few times. (wontfix for now)
* Small thing, but `Hispanic/Latino` is down there on race. Is there a better way to rep this? (wontfix)

**Comment by verbingthenoun** [comment]
Alexa here, adding a couple of mine:

- I couldn't always get the clinic name to show up in the pledge popup, even after refreshing.

- Maybe more of an issue when doing data entry than real-time call logging, but not being able to update the initial call date was a problem if entered incorrectly, since LMP dates all work off of that call date.

- It didn't appear that it was possible to create two separate entries for the same patient. I know from past work with the CM spreadsheets that we occasionally have one patient with two completely different cases, and I'm not sure how we would handle that in this system.

**Comment by NerdyGirl537** [comment]
For the import, I grabbed the file early and did a scrub for repeats.  We're still having problems with that as some CMs don't go and find the original entry and others maintain the PT during their shift and then update the old one as well (very confusing).  Also, CMs are putting in comments on spelling for the patient names (ex  "Sp?" or "Not sure on spelling" or just "(?)").

On the initial call date, I did enter one or two wrong and wondered about how that will affect the patient record.  Agree with Alexa that it may just be on the data entry side.

We probably need to look at breaking out the Planned Parenthood clinics to individual ones (e.g. PPMW includes Silver Springs, Suitland, Falls Church and Gaithersburg) so that we can find the correct fax number for the pledges.

Gobs and gobs of data is only being stored in the notes section and not in specified cells -- appointment date, clinic information, procedure cost.

Adults/Minors in Household doesn't move over well.  Sometimes info in the notes would help, but often the spreadsheet just had a number.

The patient info screen doesn't refresh at all.  Can go from new patient to pledge sent and status will remain "Needs Appointment".

Ended up with a patient from the Virginia line in the DC XLS that wasn't obvious until I went to enter the notes.  Not sure what's to be done on that.
